File: twidere/src/main/java/org/mariotaku/twidere/view/holder/TwoLineWithIconViewHolder.java
/*
 * 				Twidere - Twitter client for Android
 * 
 *  Copyright (C) 2012-2014 Mariotaku Lee <mariotaku.lee@gmail.com>
 * 
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 * 
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU General Public License for more details.
 * 
 *  You should have received a copy of the GNU General Public License
 *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

package org.mariotaku.twidere.view.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.CheckBox;
import android.widget.ImageView;
import android.widget.TextView;

public class TwoLineWithIconViewHolder extends RecyclerView.ViewHolder {

    public final ImageView icon;
    public final TextView text1, text2;
    public final CheckBox checkbox;

    public TwoLineWithIconViewHolder(final View itemView) {
        super(itemView);
        icon = (ImageView) itemView.findViewById(android.R.id.icon);
        text1 = (TextView) itemView.findViewById(android.R.id.text1);
        text2 = (TextView) itemView.findViewById(android.R.id.text2);
        checkbox = (CheckBox) itemView.findViewById(android.R.id.checkbox);
    }
}

File: twidere/src/main/kotlin/org/mariotaku/twidere/view/holder/TwoLineWithIconViewHolder.kt
/*
 *             Twidere - Twitter client for Android
 *
 *  Copyright (C) 2012-2017 Mariotaku Lee <mariotaku.lee@gmail.com>
 *
 *  This program is free software: you can redistribute it and/or modify
 *  it under the terms of the GNU General Public License as published by
 *  the Free Software Foundation, either version 3 of the License, or
 *  (at your option) any later version.
 *
 *  This program is distributed in the hope that it will be useful,
 *  but WITHOUT ANY WARRANTY; without even the implied warranty of
 *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 *  GNU General Public License for more details.
 *
 *  You should have received a copy of the GNU General Public License
 *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */

package org.mariotaku.twidere.view.holder

import androidx.recyclerview.widget.RecyclerView
import android.view.View
import android.widget.CheckBox
import android.widget.ImageView
import android.widget.TextView

open class TwoLineWithIconViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    val icon: ImageView = itemView.findViewById<ImageView>(android.R.id.icon)
    val text1: TextView = itemView.findViewById<TextView>(android.R.id.text1)
    val text2: TextView = itemView.findViewById<TextView>(android.R.id.text2)
    val checkbox: CheckBox? = itemView.findViewById<CheckBox>(android.R.id.checkbox)

}
